addons/hi-res-thumbnails/addon.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Higher resolution thumbnails",
+  "description": "Replaces thumbnails across the website with higher resolution images. Useful on retina displays or browsing at >100% zoom.",
+  "info": [
+    {
+      "type": "warning",
+      "text": "Higher resolution images require larger filesizes. Take care not to accidentally incur data expenses if your network plan is limited.",
+      "id": "imagefilesize"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Dimension Multiplier (%)",
+      "id": "multiplier",
+      "type": "integer",
+      "min": 100,
+      "max": 300,
+      "default": 200
+    },
+    {
+      "name": "Fix initially low-res thumbnails",
```

shouldn't be a setting imo

but if you really want to keep it,
```suggestion
      "name": "Resize initially undersized thumbnails",
```

also, mxmou made an exhaustive list of all sizes of images here fyi: https://github.com/LLK/scratch-www/issues/5598#issuecomment-864118903